lib/yuriita/exclusive_collection.rb

```diff
@@ -43,13 +43,17 @@ def params(option)
 
     def build_query(option)
       if selected?(option)
-        query.dup
+        option_query = query.dup
```

Comment:
There is a small change in behavior here. Previously, for the exclusive menu behavior, when someone selected an already selected menu item there was no change.

This changes the behavior to remove the selected item from the input string. If the selected item was the default item, it will be re-selected automatically, but the input will remain omitted from the query string. This behavior matches GitHub's. You can see this behavior on GitHub in the Sort menu. Try selecting and unselecting the default sort option. This can also be viewed in our own Sort menu on the movies page.